components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/tests/Storage/PublishedNodesJobConverterTests.cs

```diff
@@ -231,7 +231,7 @@ private class StandaloneIdentity : IIdentity {
             Assert.Equal("testfieldid1", jobs
                 .Single().WriterGroup.DataSetWriters
                 .Single().DataSet.DataSetSource.PublishedVariables.PublishedData.First().Id);
-            Assert.Equal("i=2259", jobs
+            Assert.Equal(null, jobs
```

Comment:
Doesn't this invalidate the test itself ? Can we please check that `i=2258` and `i=2259` nodes are still present in `jobs` ?

Reply:
yes, that would make sense